.travis.yml

```diff
@@ -70,3 +77,6 @@ matrix:
     - <<: *_flutter_job_template
       name: Build Flutter
       stage: flutter
+    - <<: *_flutter_example_job_template
+      name: Test Flutter Example
+      stage: flutter_example
```

Comment:
This stage is not declared in `stages` on top of the file.

Reply:
Missed that. Fixed.